File: PYTHON/Questões Uri/1036.py
lenha = input().split()
a, b, c = lenha

a = float(a)
b = float(b)
c = float(c)

if a == 0.0  or (b ** 2 - 4 * a * c) < 0:
    print('Impossivel calcular')
else:
    x1 = (- b + (b ** 2 - 4 * a * c) ** (1/2) )/(2 * a)
    x2 = (- b - (b ** 2 - 4 * a * c) ** (1/2) )/(2 * a)
    print("R1 = %0.5f"%x1)
    print("R2 = %0.5f"%x2)

File: ESTUDO DE LINGUAGENS/JAVASCRIPT/Questões uri/1036.js
const lenha = (line) => line.split(" ").map(a => +a)
const [a,b,c] = lenha(lines.shift())
let x
let y

if (a === 0 || (((b**2)-4*a*c)<0)){
    console.log("Impossivel calcular")
}else{
    x=(-b +((b*b)-4*a*c)**(1/2))/(2*a)
    y=(-b -((b*b)-4*a*c)**(1/2))/(2*a)
    console.log(`R1 = ${x.toFixed(5)}`)
    console.log(`R2 = ${y.toFixed(5)}`)
}